packages/qwik/src/core/signal/signal-api.ts

```diff
@@ -1,12 +1,33 @@
 import type { QRLInternal } from '../shared/qrl/qrl-class';
 import type { QRL } from '../shared/qrl/qrl.public';
-import { ComputedSignal, Signal, throwIfQRLNotResolved } from './signal';
+import {
+  ComputedSignalImpl,
+  SerializerSignalImpl,
+  Signal as SignalImpl,
```

Comment:
should we change name of the Signal class to SignalImpl isntead just renaming import? (to be consistent)